pages/MusicFileDetail.vue

```diff
@@ -74,52 +75,40 @@
 
 <script>
 import store from '../store';
+import { mapGetters } from 'vuex'
 
 export default {
-    asyncData({ $axios }) {
-        // AxiosによるHTTP通信 ...（1）
-        // $axios.$get(`api/${this.$store.state.auth.user.id}/follow`)
-        // .then(response => {
-        //     this.followInfo = response
-        //     this.followingId = this.followInfo.followInfo[0].following_id
-        // })
-    },
     data(){
         return {
             clickedFileUserId :'',
             clickedLoginUserId :'',
             followInfo: [],
             followingId: '',
+            followedId: '',
         }
     },
-    beforeCreate: function() {
+    asyncData: function(context) {
         let clickedFileUserId = ''
-        this.$store.getters['musicFiles/musicFileData'].forEach(musicFiledatum => {
+        context.store.getters['musicFiles/musicFileData'].forEach(musicFiledatum => {
             clickedFileUserId = musicFiledatum.clickedFileUserId
-            // console.log(musicFiledatum.clickedFileUserId)
         });
-        // console.log(clickedFileUserId)
-        // console.log(this.$store.getters['musicFiles/musicFileData'])
-        this.$axios.$get(`api/${clickedFileUserId}/${this.$store.state.auth.user.id}/getFollowInfo`)
-        .then(response => {
-            this.followInfo = response
-            this.followingId = this.followInfo.followInfo[0].following_id
-            // console.log(response)
+        context.store.dispatch('musicFiles/setFollowState', {
```

Comment:
こちらに非同期処理を入れるといかがでしょうか？
以下のイメージです。
```suggestion
       await context.store.dispatch('musicFiles/setFollowState', {
```